pyravendb/connection/requests_executor.py

```diff
@@ -170,6 +179,10 @@ def first_topology_update(self, initial_urls):
                 self.update_topology(ServerNode(url, self._database_name))
                 self.update_topology_timer = Utils.start_a_timer(60 * 5, self.update_topology_callback, daemon=True)
                 return
+            except exceptions.DatabaseDoesNotExistException as e:
+                if len(initial_urls) == 1:
```

Comment:
Should always raise this, the exception is not recoverable

Reply:
What happens if you have more the one url in the store, and in one of those url the database exists?

Reply:
Not possible, the database topology in in Raft, so it is consistent.

Reply:
Every node in the cluster will know to tell you where to go there.

Reply:
You are right if this is the case, but what if the user do something else/
for example 2 clusters . or give you just 2 address of different servers?
If it wrong way to use this so yes you need to throw right way